### ISSUE TITLE
Error LNK2019 unresolved external symbol _main referenced in function "int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ)

### ISSUE THREAD

**Issue by fmnijk** [HypatiaOfAlexandria/MortalClient#4]
When I build in Windows NT. I get this error. I tried all the answers I can find on Google. No one can work. Need help.

I change `add_executable(JourneyClient ${SOURCE_FILES})` to `add_executable(JourneyClient WIN32 ${SOURCE_FILES})` in `CMakeLists.txt`. The issue change, but it still there.

The issue change from:
`msvcrtd.lib(exe_main.obj) : Error LNK2019 unresolved external symbol _main referenced in function "int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ)`
To:
`msvcrtd.lib(exe_winmain.obj) : Error LNK2019 unresolved external symbol _winmain referenced in function "int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ)`

**Comment by fmnijk** [comment]
I guess I know why. It is because it cannot find the `main` or the `WinMain` function.

I found the `main` function in `Journey.cpp`, but I don't know how to fix the issue.

**Comment by fmnijk** [comment]
I found the answer.
Instead of searching the solutions for the LNK2019 error. I try to figure out why Visual Studio didn't see the `main` function.

Then I saw this:https://stackoverflow.com/questions/13871617/winmain-and-main-in-c-extended
I guess VS is looking for different name's main function. Likes `WinMain` or `wmain`.

Hence I tried all of these:`main`, `Main`, `wmain`, `wMain`, `WinMain`, `wWinMain`.
Then I found the exactly one and build successfully.

To solve this issue. Change `main` function in `Journey.cpp` into `wmain` function. Or copy it and make `main` and `wmain` exist together.
